blobstore/blobstore.go

```diff
@@ -106,6 +111,41 @@ func handler(r *data.FileRepo) http.Handler {
 func main() {
 	defer shutdown.Exit()
 
+	usage := `
+usage: flynn-blobstore <command> [<args>...]
+
+Commands:
+	help        show usage for a specific command
+	cleanup     delete file blobs from default backend 
+	migrate     move file blobs from default backend to a different backend
```

Comment:
I default that to the absence of a command (i.e. `command == ""`). Is that not ok?